test/kafka/manager/TestKafkaManager.scala

```diff
@@ -91,7 +91,7 @@ class TestKafkaManager extends CuratorAwareTest {
     val future = kafkaManager.getBrokerList("dev")
     val result = Await.result(future,duration)
     assert(result.isRight === true)
-    assert(result.toOption.get.nonEmpty === true)
+    assert(result.toOption.nonEmpty === true)
```

Comment:
Since this code can't compile, and nonEmpty is a member of Option, directly call noEmpty will sounds more scala style.
[error] /home/travis/build/yahoo/kafka-manager/test/kafka/manager/TestKafkaManager.scala:94: value nonEmpty is not a member of kafka.manager.BrokerListExtended
[error]     assert(result.toOption.get.nonEmpty === true)